field/arena.go

```diff
@@ -750,7 +750,7 @@ func (arena *Arena) sendDsPacket(auto bool, enabled bool) {
 		dsConn := allianceStation.DsConn
 		if dsConn != nil {
 			dsConn.Auto = auto
-			dsConn.Enabled = enabled && !allianceStation.Estop && !allianceStation.Astop && !allianceStation.Bypass
+			dsConn.Enabled = enabled && !allianceStation.Estop && !(auto && allianceStation.Astop) && !allianceStation.Bypass
```

Comment:
is this a prior bug that allowed Astop to be pressed during non-autonomous section ?